contracts/BridgeVault.sol

```diff
@@ -49,7 +49,6 @@ contract BridgeVault is Ownable, IBridgeVault {
         // Unwrap the WETH
         wETH.withdraw(amount);
 
-        // TODO: check transfer success
```

Comment:
don't we need to check transfer success or the following line will revert?

Reply:
both `address.tranfser` and `wETH.withdraw` either succeeds or reverts so its not necessary